be/src/exec/mor_processor.h

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <atomic>
+
+#include "exec/hash_joiner.h"
+#include "exprs/expr_context.h"
+#include "runtime/descriptors.h"
+#include "util/runtime_profile.h"
+
+namespace starrocks {
+
+struct HdfsScannerParams;
+class MorProcessor {
+public:
+    MorProcessor() = default;
+    virtual ~MorProcessor() = default;
+
+    virtual Status init(RuntimeState* runtime_state, const HdfsScannerParams& scanner_params) { return Status::OK(); }
```

Comment:
is there ay way not to bind hdf scanner params to this class?

Reply:
done